autogen/oai/client.py

```diff
@@ -319,13 +324,7 @@ def _completions_create(self, client, params):
                         ),
                     )
                 )
-        else:
-            # If streaming is not enabled or using functions, send a regular chat completion request
-            # Functions are not supported, so ensure streaming is disabled
-            params = params.copy()
-            params["stream"] = False
-            response = completions.create(**params)
-        return response
```

Comment:
Removing this branch doesn't look right. When streaming is not enabled, we still need this branch.
Removing "else" is OK because the if branch will always return.

Reply:
Oh, you're right @sonichi sorry, I'll add that back. So essentially there are 3 use-cases we're covering
1. Streaming is enabled, and no function calls are being made
2. Streaming is disabled (in which case it doesn't matter whether function calls are being made or not)
3. Streaming is enabled, and functions calls ARE being made, in which case we would like agents to be able to utilise streaming when sending messages, however, function calls (for the meantime) do not stream